app/build.gradle

```diff
@@ -208,7 +208,7 @@ dependencies {
     implementation 'com.github.TeamNewPipe:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
     // WORKAROUND: if you get errors with the NewPipeExtractor dependency, replace `v0.24.3` with
     // the corresponding commit hash, since JitPack is sometimes buggy
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:v0.24.4'
+    implementation 'com.github.FireMasterK:NewPipeExtractor:d2cbd09089e8af933738f98b671ad58236a79d6e'
```

Comment:
Oh I missed that the NewPipeExtractor still points to an unofficial repo, that needs to be fixed before merge